### ISSUE TITLE
Editor top bar has preview, update, launch options. We need to get rid of "Update".

### ISSUE THREAD

**Issue by olaolusoga** [Automattic/wp-calypso#42588]
We don't need update

![Screen Shot on 2020-05-22 at 12-47-27](https://user-images.githubusercontent.com/1382943/82690788-dbe50400-9c2a-11ea-9529-122ead860bbf.png)

**Comment by p-jackson** [comment]
I'm confused about not needing update. I might be missing some extra context about a new save feature that's coming. Is there a link to some discussion?

1. Land on home page editor after `/new` flow
2. Make changes to the lorem text, I won't want to launch that
3. Can't find way to save doc
4. Click (W), get unsaved work prompt
  - Will lose work if I click OK
  - Stuck in a loop if I click Cancel

@ramonjd

**Comment by simison** [comment]
I agree, we shouldn't just remove it.

Until the site is launched, we should either:

- Rename it to "Save" (I know previous attempt failed)
- Remove it and have the document save continuously, no separate saving needed.

**Comment by ramonjd** [comment]
I threw up https://github.com/Automattic/wp-calypso/pull/42608 without much regard to the consequences. 

I'm closing that PR until we come up with a way to address the UX deficit .

Thanks for the input!

**Comment by olaolusoga** [comment]
Sorry, I should have added more context to issue.

The action saves the doc. Update feels incorrect. We should relabel to "Save".

>Rename it to "Save" (I know previous attempt failed)

What failed?

**Comment by ramonjd** [comment]
> Rename it to "Save" (I know previous attempt failed)

(I think) React had a major hissy fit when it couldn't reconcile what it considered to be the right DOM node tree. 

We messed around with a text node, and when it couldn't find it, it threw its toys in the corner and went home, which is a metaphor for throwing an exception and borking the whole site.

https://github.com/Automattic/wp-calypso/pull/42508

**Comment by ramonjd** [comment]
Placing on hold until we work out what the best solution would be here. 

We can't mess with the text area (See: #42508). 

Adding a button is possible, we do this with **Launch**, but there's a [move to relocate that to FSE ](https://github.com/Automattic/wp-calypso/issues/42647)

**Comment by ollierozdarz** [comment]
This ticket is 5 years old and most likely irrelevant now. I'm in favour of it being closed?

**Comment by ramonjd** [comment]
Agree. I closed it. Then opened it. Then closed it again (as not planned now 😄). 

We can always reopen if necessary.
